components/src/components/SimpleContent/Component.ts

```diff
@@ -31,4 +31,17 @@ export default class Component extends (ParentComponent as any) {
             schema: Component.schema()
         };
     }
+
+    addTargetBlankToLinks = (html) => {
+      let result = '';
+      if(html.includes('<a href=')) {
+        result = html.replace(/<a href=/g, '<a target="_blank" href=');
+      }
+      return result;
+    };
+
+    constructor(component, options, data) {
+      super(component, options, data);
+      component.html = this.addTargetBlankToLinks(component.html);
+    }
```

Comment:
Seems to have broken the Static Text control? Does it work on your local. Anything I enter isn't saved any more on this control on the PR.

Reply:
Unintentional add from my commit. This has been reverted now.